docs/customization_guide/inference_protocols.md

```diff
@@ -479,7 +479,7 @@ JavaCPP-presets. You can do this using the following steps:
    For C-API Wrapper Java bindings, you need to install some build specific dependencies like cmake. You can either do this with:
    ```bash
     $ git clone https://github.com/triton-inference-server/developer_tools.git
-    $ bash -x ../install_test_dependencies_and_build.sh
+    $ bash -x developer_tools/server/install_dependencies_and_build.sh
```

Comment:
I don't think the script lives at this path in the dev tools repo, does it?

Reply:
Oh nvm, I see you moved it in the other PR

Reply:
>    For C-API Wrapper Java bindings, you need to install some build specific dependencies like cmake. You can either do this with:

Not sure I understand, isn't the C API Java bindings script this one: https://github.com/triton-inference-server/client/blob/main/src/java-api-bindings/scripts/install_dependencies_and_build.sh, not the script in the dev tools repo?

Reply:
There are 2 java bindings. The client repo contains the entire build script for *java* bindings. The developer tools repo contains build instructions for *C++ bindings*. This does not include anything related to java.

Reply:
But to make it clear I'll just point people to the client script which contains build steps for both versions of the java bindings